files/rollup.config.mjs

```diff
@@ -60,7 +63,11 @@ export default {
     addon.gjs(),<% if (typescript) { %>
 
     // Emit .d.ts declaration files
-    addon.declarations('declarations'),<% } %>
+    addon.declarations(
+      'declarations',<% if (packageManager === 'pnpm') { %>
+      `pnpm glint --declaration --project ${configs.ts}`,<% } else { %>
```

Comment:
That line does not work. I tested it with `glint@1.5` and `unstable`. This command will still use `tsonfig.json` (not the publish one).

Reason being, the `--project` parameter resolves to a directory in which it searches for the nearest tsconfig file. 

1. https://github.com/typed-ember/glint/blob/1.5.2/packages/core/src/config/index.ts
2. https://github.com/typed-ember/glint/blob/5231f68d55ac9a27cbe3849a92b0d597c9e2b3f9/packages/core/src/config/loader.ts#L31-L48

The code is the same for `1.5` as well as for `unstable` glint. That's a bug in glint I'd say?!